articles/user-guide/testing-debugging.md

```diff
@@ -379,16 +408,16 @@ namespace app
     operation Operation () : Unit {
 
         use qubits = Qubit[2];
-	X(qubits[1]);
-	H(qubits[1]);
-	R1Frac(1, 2, qubits[1]);
+        X(qubits[1]);
+        H(qubits[1]);
+        R1Frac(1, 2, qubits[1]);
+
+        DumpMachine("dump.txt");
+        DumpRegister("q0.txt", qubits[0..0]);
+        DumpRegister("q1.txt", qubits[1..1]);
 
-	DumpMachine("dump.txt");
-	DumpRegister("q0.txt", qubits[0..0]);
-	DumpRegister("q1.txt", qubits[1..1]);
+        ResetAll(qubits);
 
```

Comment:
This empty line also looks like an artifact of using -> use migration

Reply:
True that could be the case. I thought it was to provide symmetry to the new line at the start of the function body. Leaving it at the top but removing the bottom one looks strange to me, but I'm happy to change it to whatever is considered best style.
Unfortunately the formatting of our code snippets and samples is not very consistent for things like these, as the Q# style guide doesn't cover them.